protocols/bgp/server/fsm_open_sent.go

```diff
@@ -137,6 +137,7 @@ func (s *openSentState) openMsgReceived(openMsg *packet.BGPOpen) (state, string)
 
 func (s *openSentState) handleOpenMessage(openMsg *packet.BGPOpen) (state, string) {
 	s.fsm.holdTime = time.Duration(math.Min(float64(s.fsm.peer.holdTime), float64(time.Duration(openMsg.HoldTime)*time.Second)))
+	fmt.Printf("s.fsm.holdTime: %d\n", s.fsm.holdTime)
```

Comment:
in my opinion, this should be logging, but can be fixed later

Reply:
Fixed it.